Onboard/OnboardGtk.py

```diff
@@ -568,6 +568,8 @@ def cb_keys_changed(self, keymap):
     def cb_keyman_changed(self, name):
         """ keyman keyboard change """
         print("keyman changed to", name)
+        # TODO: why is 'None' passed as a string here???
```

Comment:
Do we need to track this?

Reply:
https://github.com/keymanapp/keyman/issues/12330